scripts/src/java/org/oppia/android/scripts/ci/ComputeAffectedTests.kt

```diff
@@ -103,10 +103,18 @@ private fun computeAffectedTargetsForNonDevelopBranch(
   println("Affected test targets due to transitive build deps: $transitiveTestTargets")
 
   val allAffectedTestTargets = (affectedTestTargets + transitiveTestTargets).toSet()
+
+  // Filtering out the targets to be ignored.
+  val filteredAffectedTestTargets = allAffectedTestTargets.filter { file ->
+    !file.startsWith("//instrumentation", ignoreCase = true)
+  }
```

Comment:
@FareesHussain The approach looks correct to me!
Just a nit suggestion! 
Instead of `file` in the lambda can we use it as `filePath` for a better context to what it refers to?

Reply:
@Sparsh1212 its more of a `target` I'll use that

Reply:
Done